File: src/codes/Users_Manual/sampling_config/sampling_config_freq_nearest.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
# let s = 
SamplingConfig::FreqNearest(4000.0 * Hz)
# ;
// or
# let s = 
SamplingConfig::PeriodNearest(std::time::Duration::from_micros(250))
# ;
# }

File: src/codes/Users_Manual/sampling_config/sampling_config_freq_nearest.py
~from pyautd3 import SamplingConfig, Hz
~from datetime import timedelta
SamplingConfig.FreqNearest(4000.0 * Hz)
# or
SamplingConfig.PeriodNearest(timedelta(microseconds=250))
# or
SamplingConfig.DivisionRaw(5120)